odoo/addons/base/models/ir_ui_view.py

```diff
@@ -606,7 +606,10 @@ def locate_node(self, arch, spec):
     def inherit_branding(self, specs_tree, view_id, root_id):
         for node in specs_tree.iterchildren(tag=etree.Element):
             xpath = node.getroottree().getpath(node)
-            if node.tag == 'data' or node.tag == 'xpath' or node.get('position') or node.get('t-field'):
+            if node.get('t-field'):
+                node.set('data-oe-field-xpath', xpath)
+                self.inherit_branding(node, view_id, root_id)
+            elif node.tag == 'data' or node.tag == 'xpath' or node.get('position'):
```

Comment:
Alternatively, add just a conditional inside the old one to special-case the t-field?
```suggestion
            if node.tag == 'data' or node.tag == 'xpath' or node.get('position') or node.get('t-field'):
                if node.get('t-field'):
                    node.set('data-oe-field-xpath', xpath)
```

Reply:
doesn't make a big difference though

Reply:
Yes I did that at first. Was not able to decide which I found more readable, I chose the one not repeating the t-field condition. It's actually "the most efficient" (:smile:) and does not repeat more code that the other... but I change if you'd like :)

Reply:
Nah, it's fine.